### ISSUE TITLE
Warning from Temp Sensor Ford Focus

### ISSUE THREAD

**Issue by AlexW75** [itchannel/fordpass-ha#478]
Logger: homeassistant.components.sensor.recorder
Source: components/sensor/recorder.py:245
Integration: Sensor (documentation, issues)
First occurred: 23:55:10 (1 occurrences)
Last logged: 23:55:10

The unit of sensor.fordpass_outsidetemp (°C) cannot be converted to the unit of previously compiled statistics (None). Generation of long term statistics will be suppressed unless the unit changes back to None or a compatible unit. Go to https://my.home-assistant.io/redirect/developer_statistics to fix this

**Comment by SquidBytes** [comment]
Yes, I have this error too. I believe I messed up when I added that sensor (or attribute) to the code. I've been looking into it but its not a priority - and I have very minimal time
